### ISSUE TITLE
upload sources and prepare readme

### ISSUE THREAD

**Issue by chaos-adept** [spaceshipyard/mars-rover-virtual-bridge#1]
@vlfors  this repository is empty now,

Could you upload actual files and write short readme how to start?

**Comment by vlfors** [comment]
I uploaded project, but it still needs in refactoring.

**Comment by chaos-adept** [comment]
@vlfors 
it have tried to do steps in the readme, i have faced the issue with command
```
python setup.py install
```

```
D:\projects\mars\mars-rover-virtual-bridge>python setup.py install
Traceback (most recent call last):
  File "setup.py", line 1, in <module>
    from setuptools import setup, find_packages
ModuleNotFoundError: No module named 'setuptools'
```

i have tried to reinstall python 3.5.6 (i have 2.7 in the same time) but it does not work for me

**Comment by vlfors** [comment]
Need install setuptools  
https://pypi.org/project/setuptools/
command:
python -m pip install --upgrade pip setuptools wheel

PS. I'm sorry, just today noticed question.

**Comment by chaos-adept** [comment]
the issue was resolved by removing python 2.7 from the PC and installing Python 3.6 from windows installer
